dashboard/views.py

```diff
@@ -625,6 +641,57 @@ def are_weighted_assignments_hidden(course_id, df):
             return True
 
 
+def find_binning_grade_value(grades):
+    fifth_item = grades[4]
+    next_to_fifth_element = grades[5]
+    if next_to_fifth_element - fifth_item > 2:
+        binning_grade_value, index = fifth_item, 4
+    else:
+        binning_grade_value, index = binning_logic(grades, fifth_item)
+    return binning_grade_value, index
+
```

Comment:
@jonespm  had me start using `namedtuple` when returning tuples from functions in [this PR](https://github.com/tl-its-umich-edu/my-learning-analytics/pull/652).